setup.py

```diff
@@ -10,5 +10,12 @@
             "python-openid ; python_version<'3'",
             "python3-openid ; python_version>='3'",
         ],
+        'test': [
```

Comment:
The tests all run inside `tox`, which defines its own lust of dependencies. I'm not sure if we should be adding these here - it would be better to just make sure that `python setup.py test` ran `tox`.